.github/workflows/publish.yml

```diff
@@ -5,26 +5,40 @@ permissions:
 jobs:
   publish:
     runs-on: ubuntu-latest
+    env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
```

Comment:
Do we need this one?

Reply:
We do need the github token, I'm going to see if we can do without node auth token but I think that it is used internally in some of the actions